src/granite/query/assemblers/base.cr

```diff
@@ -120,16 +120,20 @@ module Granite::Query::Assembler
                     end
     end
 
-    def count : Executor::Value(Model, Int64)
+    def count(use_group_by : Bool = false) : (Executor::MultiValue(Model, Int64) | Executor::Value(Model, Int64))
       sql = build_sql do |s|
         s << "SELECT COUNT(*)"
         s << "FROM #{table_name}"
         s << where
-        s << group_by
+        s << group_by if use_group_by
```

Comment:
I think you should remove the `group_by` for now.  This shouldn't be automatically adding a group by unless someone specifies it.  Not sure why this was added to begin with.